README.md

```diff
@@ -1,20 +1,38 @@
-# Accessibility Check Action
+# accessibility-check-action
 
-In order for Stark to scan your GitHub web repository, you’ll need to set up a custom workflow. This workflow enables Stark to scan your web repository for accessibility issues.
+## Table of Contents
 
-## Setting up your repository
+- [Basic Usage](#basic-usage)
+- [Usage with Stark Projects](#usage-with-stark-projects)
+- [More Examples](#more-examples)
+- [Technical Details](#technical-details)
 
-First, create a Github workflow file at `.github/workflows/starkflow.yml`. (The file name is important: that’s how Stark finds the correct workflow to run.)
+## Basic Usage
+
+Basic usage of the action is exactly as shown in the next section's template, except you don't need to provide the action a `token`.
+
+## Usage with Stark Projects
```

Comment:
Thoughts on swapping this with the section above? So it's Usage with Stark Projects first? Or perhaps changing the first section to Usage instead of Basic Usage and calling this section "Connecting with Stark Projects"